helm/superset/README.md

```diff
@@ -144,6 +145,7 @@ helm install my-superset superset/superset
 | supersetCeleryFlower.service.annotations | object | `{}` |  |
 | supersetCeleryFlower.service.port | int | `5555` |  |
 | supersetCeleryFlower.service.type | string | `"ClusterIP"` |  |
+| supersetCeleryFlowerservice.nodePorts.http | int | `""` |  |
```

Comment:
Were these added manually? Pls re-generate this file using `helm-docs`

Reply:
Updated, thanks!